client/trino-client/src/test/java/io/trino/client/TestIntervalDayTime.java

```diff
@@ -51,10 +52,10 @@ public void textMaxDays()
         assertEquals(toMillis(days, 0, 0, 0, 0), DAYS.toMillis(days));
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class)
+    @Test
     public void testOverflow()
     {
         long days = (Long.MAX_VALUE / DAYS.toMillis(1)) + 1;
-        toMillis(days, 0, 0, 0, 0);
+        assertThatThrownBy(() -> toMillis(days, 0, 0, 0, 0)).isInstanceOf(IllegalArgumentException.class);
```

Comment:
can you add a line break before `.isInstanceOf` and also verify the exception message with `.hasMessage("some expected message");`?

```
assertThatThrownBy(...)
    . isInstanceOf(....)
    .hasMessage(....);
```